test/ig/IGExporter.IG.test.ts

```diff
@@ -1913,6 +1913,10 @@ describe('IGExporter', () => {
       loggerSpy.reset();
       tempOut = temp.mkdirSync('sushi-test');
       config = cloneDeep(minimalConfig);
+      config.menu = [
+        { name: 'Home', url: 'index.html' },
+        { name: 'Artifacts', url: 'artifacts.html' }
+      ];
```

Comment:
Since you do this six times, consider making a `minimalConfigWithMenu` instead -- then just use that instead of `minimalConfig` in these places. That way you only need to specify the config menu in one place. It's a small thing, but I feel like it's a little cleaner that way.